### PR TITLE
Channels: Larger fields, better error handling

### PR DESCRIPTION
**Pull request by annando**
The tag fields and the fulltext search field is now larger. Also These values are now editable in a textarea. The tag fields are also sanitized. 

Errors in the channel definition are now caught and displayed.

### CODE REVIEW

File: src/Module/Conversation/Timeline.php [hunk 1]
```diff
@@ -346,6 +346,10 @@ private function getRawChannelItems()
 
 		$items = [];
 		$result = $this->database->select('post-engagement', ['uri-id', 'created', 'owner-id', 'comments', 'activities'], $condition, $params);
+		if ($this->database->errorNo()) {
+			throw new \Exception($this->database->errorMessage(), $this->database->errorNo());
```
Comment (MrPetovan) [inline src/Module/Conversation/Timeline.php:350]: Shouldn't we do this from the database library itself?

Comment (annando) [reply src/Module/Conversation/Timeline.php:350]: I don't want to anything at the database level, because that would had a really heavy impact.

Comment (MrPetovan) [reply src/Module/Conversation/Timeline.php:350]: Hm...

**Review by MrPetovan — APPROVED** [review]

**Comment by SpcCw** [comment]
@annando - what is "access key" function? I don't get how it works so I am not sure how translate it correctly.

**Comment by MrPetovan** [comment]
It's a keyboard shortcut to activate the link to the relevant channel. More info on the `accesskey` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/accesskey

**Pull request merged**